src/api/app/controllers/webui/main_controller.rb

```diff
@@ -21,7 +21,7 @@ def gather_busy
   end
 
   def index
-    @status_messages = StatusMessage.alive.limit(4).to_a
+    @status_messages = StatusMessage.alive.includes(:user).limit(4).to_a
```

Comment:
The `includes(:user)` can also be moved into the `alive` scope.

Reply:
Not every time the scope is used it will need to include the user. But you are right I should fix the other places where we need to prefetch the users as well.

Reply:
Fine. :+1: